pkg/api/handlers/utils/containers.go

```diff
@@ -91,25 +93,33 @@ func WaitContainer(w http.ResponseWriter, r *http.Request) (int32, error) {
 		Error(w, "Something went wrong.", http.StatusBadRequest, errors.Wrapf(err, "Failed to parse parameters for %s", r.URL.String()))
 		return 0, err
 	}
-
-	if len(query.Condition) > 0 {
-		UnSupportedParameter("condition")
+	interval, err := time.ParseDuration("250ms")
+	if err != nil {
+		InternalServerError(w, err)
+		return 0, err
+	}
```

Comment:
Not a biggy, but could you do this ParseDuration as an else for line 101?

Reply:
@rhatdan hates elses

Reply:
yeah, but without one here you're doing ParseDuration twice if you get a value fed in.  Your call.

Reply:
ok, sure!